ghost-gcs-backup/script/do-backup.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
I don't think this file is needed anymore :)

Reply:
Indeed!